src/main/frontend/app/containers/BarFormContainer.js

```diff
@@ -0,0 +1,133 @@
+import React, {Component} from 'React'
+import TextField from '../components/TextField'
+import NumberField from '../components/NumberField'
+import SelectField from '../components/SelectField'
+import TextAreaField from '../components/TextAreaField'
+
+class BarFormContainer extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      barName:'',
+      description:'',
+      location:'',
+      barOwner:'',
+      beachOptions:['Near the Beach','Far from the Beach'],
+      beachText:'Far from Beach',
+      hasBeach:false
+    }
+    this.handleNameInput = this.handleNameInput.bind(this)
+    this.handleDescriptionInput = this.handleDescriptionInput.bind(this)
+    this.handleLocationInput = this.handleLocationInput.bind(this)
+    this.handleBarOwnerInput = this.handleBarOwnerInput.bind(this)
+    this.handleBeachOptionsInput = this.handleBeachOptionsInput.bind(this)
+    this.handleClearForm = this.handleClearForm.bind(this)
+    this.handleSubmit = this.handleSubmit.bind(this)
+  }
+
+//handleChange function
```

Comment:
please remove comments before submitting PRs

Reply:
There are several in this file. I am not going to comment on all of them, but please check the whole PR